### ISSUE TITLE
2 buttons in About dlg are not aligned

### ISSUE THREAD

**Issue by Alexey-T** [doublecmd/doublecmd#704]
### Version

Double Commander
Version: 1.0.9
Revision: 222
Commit: a844456e4
Build date: 2022/12/03
Lazarus: 2.2.2.0
FPC: 3.2.2
Platform: x86_64-Linux-gtk2
OS version: Ubuntu 22.04.1 LTS

Widgetset library: GTK 2.24.33

### Description

Dialog: 2 buttons 'copy to clipbd' and 'close' must have the same height and width.

![ab](https://user-images.githubusercontent.com/4154978/206819758-b8f7a995-0f1e-4027-90a8-78dcb8b985a1.png)

**Comment by alexx2000** [comment]
They aligned by center.

**Comment by Alexey-T** [comment]
they should have the same height and width.

**Comment by JonnyTech** [comment]
@Alexey-T Ui design is a personal choice. Using opinionated and aggresive wording such as "they **must**" and "they **should**" is a rather arrogant and rude thing to tell someone what to do with their project.

**Comment by Alexey-T** [comment]
yes, it is designer choice. Can I ask you to make buttons aligned, ie with the same w/h, please?

**Comment by Skif-off** [comment]
> ie with the same w/h

Maybe I'm wrong, but as far as I understand, these buttons are not equivalent and do not have to match. For example, go to "Configuration" in the main menu > Options... > Archivers: Add, Rename or Delete vs. OK, Cancel, Apply.
This seems to be the reason :)

**Comment by JonnyTech** [comment]
I too see no reason why the buttons, which serve completely different purposes, have to be identical. If anything, it makes sense for them to differ. But that is my opinion. It should be up to Alex to decide.

**Comment by timur-g** [comment]
Please close this.
